.github/workflows/parachain.yml

```diff
@@ -15,6 +15,32 @@ on:
       - '!parachain/LICENSE'
 
 jobs:
+  fmt:
+    runs-on: ubuntu-20.04
+    env:
+      CARGO_INCREMENTAL: 0
+      RUST_BACKTRACE: 1
+      RUSTFLAGS: -C debuginfo=1
+      SKIP_WASM_BUILD: 1
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-${{ hashFiles('parachain/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          target: wasm32-unknown-unknown
+          components: rustfmt
+      - name: cargo fmt
+        run: cd parachain && cargo +nightly fmt -- --config-path rustfmt.toml && cd -
```

Comment:
I used the run option here instead of the cargo runner, because the `actions-rs/cargo@v1` fmt command doesn't work when the rust project is in a subdirectory. I tested using the `manifest-path` flag with `cargo fmt`, doesn't seem to work.